### PR TITLE
Fully remove CSP as opposed to setting custom policies

### LINKED ISSUE #292: Consider fully removing CSP instead of just loosening it
**Describe the bug**
As far as I can tell, ArmCord CSP just provides a fetch() without CSP restrictions
and loosens the policy to allow CSS imports to work.

Please consider just entirely removing CSP as most mod injectors do.
It will improve compatibility with some mods / plugins.

**To Reproduce**
1. Attempt to use a plugin that imports remote code
2. CSP intercepts it

**Expected behavior**
CSP is bypassed

**Screenshots**
image

**Desktop (please complete the following information):**
 - OS: Linux
 - Method of installation: aur
 - Version: 3.1.0

**Additional context**

### PR DESCRIPTION
**Pull request by yellowsink**
Closes #292

### CODE REVIEW

File: src/extensions/mods.ts [hunk 1]
```diff
@@ -1,55 +1,18 @@
...
-            // Fix Discord's broken CSP which disallows unsafe-inline due to having a nonce (which they don't even use?)
-            csp["script-src"] = csp["script-src"]?.filter((value) => !value.includes("nonce-"));
+        const cspHeaders = Object.keys(responseHeaders!).filter((name) =>
+          name.toLowerCase().startsWith("content-security-policy")
+        );
 
-            cspHeaders[0] = stringifyPolicy(csp);
-        }
+        for (const header of cspHeaders)
+            delete responseHeaders![header];
```
Comment (Vendicated) [inline src/extensions/mods.ts]: whats the point of this filter into loop, why not just `delete responseHeaders["content-security-policy"]`?

Comment (yellowsink) [reply src/extensions/mods.ts]: Multiple `content-security-policy` headers could exist, though unlikely

Comment (yellowsink) [reply src/extensions/mods.ts]: I mean, this specific method has worked for both CC and shelter flawlessly, so I figured there was no particular cause for reworking the code now

Comment (Vendicated) [reply src/extensions/mods.ts]: there's just content-security-policy, nothing else! If u wanna be safe u can also delete content-security-policy-report-only but Discord doesnt use that

Comment (yellowsink) [reply src/extensions/mods.ts]: fair enough

**Comment by smartfrigde** [comment]
Epic

**Pull request merged**